drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -606,10 +606,7 @@ int nrf_wifi_if_start_zep(const struct device *dev)
 	       sizeof(vif_info));
 
 	vif_info.state = NRF_WIFI_FMAC_IF_OP_STATE_UP;
-
-	memcpy(vif_info.ifacename,
```

Comment:
these changes are unrelated to commit log

Reply:
Included in a separate commit